lib/widgets/action_sheet.dart

```diff
@@ -79,11 +107,22 @@ abstract class MessageActionSheetMenuItemButton extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final designVariables = DesignVariables.of(context);
     final zulipLocalizations = ZulipLocalizations.of(context);
     return MenuItemButton(
-      leadingIcon: Icon(icon),
+      trailingIcon: Icon(icon, color: designVariables.actionSheetMenuButtonForeground),
+      style: MenuItemButton.styleFrom(
+        padding: const EdgeInsets.symmetric(vertical: 12, horizontal: 16),
```

Comment:
I remember Vlad saying that he doesn't want the material "splash" animation on button presses, so let's specify that:

```dart
      style: MenuItemButton.styleFrom(
        splashFactory: NoSplash.splashFactory,
```

Then to differentiate the background color between pressed and not-pressed, it looks like we're supposed to apply 12% opacity to the `contextMenuItemBg` variable for not-pressed, and 20% opacity for pressed:
- https://www.figma.com/design/1JTNtYo9memgW7vV6d0ygq/Zulip-Mobile?node-id=3302-20443&t=3eSOW3rCFnEwnweA-0
- https://www.figma.com/design/1JTNtYo9memgW7vV6d0ygq/Zulip-Mobile?node-id=3302-20443&t=3eSOW3rCFnEwnweA-0

(Those links are for light mode, but it turns out the same values 12% and 20% are used in dark mode.)

`MenuItemButton.styleFrom` doesn't seem to support specifying different background colors based on pressed and not-pressed, but it returns a `ButtonStyle`, and `ButtonStyle` does support that. So, instead of passing `backgroundColor` to `.styleFrom`, could just add after the `styleFrom(…)`:

```dart
      )
        .copyWith(
          backgroundColor: WidgetStateColor.resolveWith((states) =>
            designVariables.contextMenuItemBg.withOpacity(
              states.contains(WidgetState.pressed) ? 0.2 : 0.12))),
```